admin/test/test_cluster_setup.py

```diff
@@ -0,0 +1,25 @@
+import mock
+
+from twisted.python.usage import UsageError
+
+from flocker.testtools import TestCase
+
+from ..acceptance import CommonOptions
+from ..cluster_setup import RunOptions
+
+
+class RunOptionsTest(TestCase):
+
+    @mock.patch.object(CommonOptions, 'postOptions')
```

Comment:
Rather than mocking I think subclassing in the test module would make sense here.

```python
class RunOptionsForTest(RunOptions):


```